src/guides/v2.3/config-guide/prod/config-reference-systemxml.md

```diff
@@ -0,0 +1,358 @@
+---
+group: configuration-guide
+title: system.xml reference
+functional_areas:
+  - Configuration
+  - System
+---
```

Comment:
If you'd like attribution for this new topic, add the following metadata:

```
contributor_name: David Lambauer
contributor_link: https://github.com/DavidLambauer
```

For an example of how this renders on the site, see the top-right corner of this topic: https://devdocs.magento.com/guides/v2.3/config-guide/log/custom-logger-handler.html

Reply:
Sure, cool thing. Thanks.